src/Nethermind/Nethermind.Blockchain/Producers/BaseBlockProducer.cs

```diff
@@ -161,7 +161,8 @@ protected virtual Block PrepareBlock(BlockHeader parent)
                 UInt256.Max(parent.Timestamp + 1, _timestamper.EpochSeconds),
                 Encoding.UTF8.GetBytes("Nethermind"))
             {
-                TotalDifficulty = parent.TotalDifficulty + difficulty
+                TotalDifficulty = parent.TotalDifficulty + difficulty,
+                Author = _sealer.Address
```

Comment:
Do we need Block.Author/Beneficiary as separate properties? One wouldn't suffice?